archapp/config.py

```diff
@@ -3,6 +3,6 @@
 """
 import os
 
-hostname = os.environ.get("ARCHAPP_HOSTNAME", "pscaa02")
+hostname = os.environ.get("ARCHAPP_HOSTNAME", "pscaa01")
```

Comment:
A note to ourselves: this will eventually also be the wrong url, if I recall the archivers moved to the s3df and Murali left this end point up because we couldn't manage to get grafana working.

Until it breaks though we won't worry about it.